.github/actions/install-dependencies/action.yml

```diff
@@ -1,5 +1,14 @@
 name: Install Dependencies
 description: Install dependencies
+inputs:
+    destination:
+        description: Location to check out rust-lightning and ldk-c-bindings repositories
+        default: ''
+        required: true
+    configureRustNightly:
+        description: Flag whether or not to install rust, which is not necessary on Ubuntu machines
```

Comment:
I believe this is necessary regardless of OS, both [macOS](https://github.com/actions/virtual-environments/blob/main/images/macos/macos-12-Readme.md#rust-tools) and [Ubuntu](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2204-Readme.md#rust-tools) runners don't have nightly Rust installed

Reply:
see swift.yml which does fine without